core/core-configuration-layer.el

```diff
@@ -329,24 +337,16 @@ refreshed during the current session."
                    (car archive) i count) t))
         (spacemacs//redisplay)
         (setq i (1+ i))
-        (unless (eq 'error
-                    (with-timeout
-                        (dotspacemacs-elpa-timeout
-                         (progn
-                           (display-warning
-                            'spacemacs
-                            (format
-                             "\nError connection time out for %s repository!"
-                             (car archive)) :warning)
-                           'error))
-                      (condition-case err
-                          (url-retrieve-synchronously (cdr archive))
-                        ('error
-                         (display-warning 'spacemacs
-                          (format
-                           "\nError while contacting %s repository!"
-                           (car archive)) :warning)
-                         'error))))
+        (if (eq 'unavailable
+                (configuration-layer-check-archive-status (car archive)))
+            (setcdr
```

Comment:
Here I am going to check fallback archives as well and if all of them unavailable - error like in first commit.